GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -199,6 +199,16 @@ public override string GVFSExecutableName
             {
                 get { return "gvfs"; }
             }
+
+            public override string ProgramLocaterCommand
+            {
+                get { return "which"; }
+            }
+
+            public override HashSet<string> UpgradeBlockingProcesses
+            {
+                get { return new HashSet<string> { "GVFS.Mount", "git", "wish" }; }

```

Comment:
Why is this a different list of processes?  Before it was shared correct and now they are separate and different than the list before this change.  Does it hurt anything to have the same list?  Also I was wondering if these `HashSet`s should be case insensitive?

Reply:
> I was wondering if these HashSets should be case insensitive?

probably! I will update this.

I will add more comments around the motivation for having different lists. The commit message includes this, but I should include it in the PR description / code comments.